frontend/src/component/project/Project/ProjectStats/ProjectStats.tsx

```diff
@@ -59,12 +65,12 @@ export const ProjectStats = ({ stats }: IProjectStatsProps) => {
             />{' '}
             <StatusBox
                 title="Features created"
-                boxText={createdCurrentWindow}
+                boxText={`${createdCurrentWindow}`}
```

Comment:
why are we doing this instaed of String(createdCurrentWindow)?

Reply:
No good reason

Reply:
Fixing it